Added a section on "Developing and Testing Webhooks" in existing page

ankitjaininfo opened the pull request:
<!-- Use this checklist to make sure your PR is ready for merge. You may delete any sections you don't need. -->

## Description of changes

Added a section "Developing and Testing Webhooks" that will help fellow developer to know how to work with webhooks and build and test local services.


## Pre-merge checklist

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Extra resources

- Current version of [Sentry webhoks docs](https://docs.sentry.io/product/integrations/integration-platform/webhooks/)

## Legal Boilerplate

<!-- Sentry employees and contractors can delete or ignore this section. -->

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

lizokm commented on src/docs/product/integrations/integration-platform/webhooks.mdx:
```diff
@@ -139,3 +141,11 @@ All webhook requests have some common elements.
+
+## Developing and Testing Webhooks
+
+If you are new to working with Sentry webhooks and wish to test the configuration and examine webhook payloads, you can utilize HTTP catch-all services. [Beeceptor](https://beeceptor.com/) is an example of such a free service, offering a designated URL where you can receive HTTP payloads and subsequently inspect the JSON payload of the event. This feature proves valuable for gaining insights and  ...
+
+Once you have reviewed payloads for the relevant event types, you can start the development of your service. Local tunnels are particularly useful for testing and debugging webhooks in a local environment before deploying them to a production server. By creating a tunnel, you can receive incoming webhook requests from Sentry to your local machine, eliminating the need for publicly accessible serv ...
```
```suggestion
If you’d like to test webhook configuration and look at payloads before starting development, an HTTP catch-all service that provides a designated URL where you can receive HTTP payloads and inspect the JSON event payload can come in handy. After you’ve reviewed the relevant event payloads, you can begin development. 

To make testing and debugging webhooks faster and easier, you can create local tunnels to get incoming webhook requests from Sentry to your local machine. There are various tools that let you pick a port or address (such as 3000 or 8080) to be tunneled, and then provide you with a temporary public URL that forwards requests to your local server.
```

lizokm reviewed (changes requested):
Hi there, thanks so much for taking the time to write this for us. We really appreciate it! We've made some edits that make this more in line with our docs writing style guide.

evanpurkhiser reviewed (approved):
Updates look and sound great to me! Thanks for getting this on our radar @ankitjaininfo

ankitjaininfo commented:
Hi @lizokm, I have incorporated all your suggestions and updated this PR.

I think you oversimplified the new section. Nevermind. Suggested for tools like beeceptor and ngrok are quite popular and shall be of great help to the reader to review payloads beforehand. I can understand that this documentation shouldn't be biased on a few tools. If you are okay, we can list these tools with a caution/warning. Let me know your thoughts, I can update the PR.
